extensions/amp-story/1.0/amp-story-page-attachment.css

```diff
@@ -125,3 +125,87 @@ amp-story[desktop] .i-amphtml-story-page-attachment-remote  {
   opacity: 1;
  }
 }
+
+/** Page attachment form submissions state styles. */
+
+.i-amphtml-story-page-attachment-form-submission-status {
+  align-items: center !important;
+  background: #FFFFFF !important;
```

Comment:
if you use `--i-amphtml-draggable-drawer-background-color` you should be able to get rid of the theme override

Reply:
Updated, although the override still seems necessary for the dark theme's border